account_move_create_payment_entry/wizards/account_payment_register.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, api, fields, models
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = "account.payment.register"
+
+    payment_entry = fields.Boolean(
+        help="Technical field to determine whether the wizard will "
+        "act as a payment entry wizard."
+    )
+
+    @api.depends("line_ids")
+    def _compute_from_lines(self):
+        super()._compute_from_lines()
+        if self.payment_entry:
+            self.can_edit_wizard = False
+        return
+
+    def _post_payments(self, to_process, edit_mode=False):
+        if self.payment_entry:
+            return
+        return super()._post_payments(to_process, edit_mode)
+
+    def _reconcile_payments(self, to_process, edit_mode=False):
+        if self.payment_entry:
+            return True
+        return super()._reconcile_payments(to_process, edit_mode)
+
+    def _create_payments(self):
```

Comment:
Shouldn't we be extending `_init_payments()` instead?

Reply:
There is no difference in functionality but I agree using `_init_payments()` is better.